package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@sso-kit/sso-kit",
```

Comment:
The `@` prefix should be the product name. I'd use `@hilla`, but let's also ask @tarekoraby if he prefers `@vaadin`.

Reply:
`@hilla` is good (since we won't use this for Flow).

Reply:
Changed the product name to `@hilla`.